source/shared/cpp/ObjectModel/SharedAdaptiveCard.cpp

```diff
@@ -105,6 +105,16 @@ std::shared_ptr<ParseResult> AdaptiveCard::Deserialize(
     std::string language = ParseUtil::GetString(json, AdaptiveCardSchemaKey::Language);
     std::string speak = ParseUtil::GetString(json, AdaptiveCardSchemaKey::Speak);
 
+    // check if language is valid
+    try 
+    {
+        std::locale(language.c_str());
+    }
+    catch (...)
```

Comment:
>(...) [](start = 10, length = 5)

can we catch a more specific exception here?

Reply:
replaced by `std::runtime_error`. Still super not specific but that's the best I could find.